extensions/identity-hub-verifier/src/main/java/org/eclipse/dataspaceconnector/identityhub/verifier/DidJwtCredentialsVerifier.java

```diff
@@ -97,8 +100,9 @@ private Result<Void> verifySignature(SignedJWT jwt, PublicKeyWrapper issuerPubli
             monitor.debug(() -> "JWT signature verification successful");
             return Result.success();
         } catch (JOSEException e) {
-            monitor.warning("Unable to verify JWT token", e);
-            return Result.failure("Unable to verify JWT token. " + e.getMessage());
+            var failureMessage = "Unable to verify JWT token";
```

Comment:
Could we try to cover this case with a unit test? We should be able to mock the `verify` method and throw an exception.